src/Microsoft.IIS.WebManager/package.json

```diff
@@ -11,7 +11,7 @@
     "@angular/router": "5.0.0",
     "angulartics2": "1.1.12",
     "bootstrap": "3.3.7",
-    "chart.js": "2.7.0",
+    "chart.js": "^2.7.2",
```

Comment:
it means at least this version. This file is automatically updated by npm to resolve some security issue.

Reply:
We underwent a security review for the IIS Administration API and Web Manager, and one of the things that came out of it was targeted toward NPM. Depending on the "latest" version of a package may result in pulling in a package that has not been verified. We did verification of packages using the npm security tool. It's a package itself that inspects node_modules to see if there are any reported vulnerabilities. I can't remember the name of the tool right now.

We explicitly versioned every package, and then reviewed each one. Once we had verified packages we generated the npm-shrinkwrap.json. Then we calculated the hashes for each one of the packages and appended them to the package entries for each package in npm-shrinkwrap.json.

This allowed us to be secure when performing npm install. We made sure to pull in only package versions we trusted, and we hashed each package we pulled and compared it with the checked in hash in npm-shrinkwrap.json. This means that the content of the package was not changed since being pulled. Although, the latest version of npm might support content verification via hash by now. Atleast I hope it does.

Reply:
npm install updated these version. I think I will revert these for now then